engine/hooks/tasks.go

```diff
@@ -315,14 +315,26 @@ func (s *Service) doWebHookExecution(t *TaskExecution) (*sdk.WorkflowNodeRunHook
 	return executeWebHook(t)
 }
 
+func getRepositoryHeader(whe *WebHookExecution) string {
+	if v, ok := whe.RequestHeader[GithubHeader]; ok && v[0] == "push" {
```

Comment:
what about create event ? https://developer.github.com/v3/activity/events/types/#createevent

Reply:
it not represents a push event

Reply:
other event will arrive after